tests/crypto_aesctr/generate_test_vectors.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+""" Generate AES-CTR test vectors with an independent implementation. """
+
+import binascii
+
+import Crypto.Util.Counter
+import Crypto.Cipher.AES
+
+PLAINTEXTS = [
+    " ", "A", "AAAA", "AB", "hello", "hello world",
+    "This is 16 chars",
+    "Ceci n'est pas 24 chars.",
+    "This block is exactly 32 chars!!",
+    ]
+
+INITIAL_VECTOR = b'\0' * 16
+
+
+def generate_for_keylen(keylen):
+    """ Print plaintext and ciphertext for the given key length.  """
+    # key: 00 02 03 04...
```

Comment:
0,2,3,4?  Is that a typo?

Reply:
Yes, a typo; the code does 0, 1, 2, etc.